docs/flow/modin/config.rst

```diff
@@ -0,0 +1,64 @@
+:orphan:
+
+Config Module Overview
+""""""""""""""""""""""
+Using this module the user can tune Modin's behavior in the way he need it. To discover
+all avaliable configs just run `python -m modin.config`, this command will print all
+Modin configs with descriptions.
+
+Public API
+''''''''''
+Potentially, the source of configs can be any, but for now only environment
+variables are implemented. Any environment variable originate from
+:class:`~modin.config.envvars.EnvironmentVariable`, which contains most of
+the config API implementation.
+
+.. autoclass:: modin.config.envvars.EnvironmentVariable
+  :members: get, put, get_help, get_value_source, once, subscribe
+
+Modin Configs List
+''''''''''''''''''
+
+.. csv-table::
+   :file: configs_help.csv
+   :header-rows: 1
+
+Usage Guide
+'''''''''''
+See example of interation with Modin configs below, as it can be seen config
+value can be set either by setting the environment variable or by using config
+API.
+
+.. code-block:: python
+
+    import os
+
+    # Setting `MODIN_BACKEND` environment variable.
+    # Also can be set outside the script.
+    os.environ["MODIN_BACKEND"] = "OmniSci"
+
+    import modin.config
+    import modin.pandas as pd
+
+    # Checking initially set `Backend` config,
+    # which corresponds to `MODIN_BACKEND` environment
+    # variable
+    print(modin.config.Backend.get())
+
+    # Checking default value of `NPartitions`
+    print(modin.config.NPartitions.get())
+
+    # Changing value of `NPartitions`
+    modin.config.NPartitions.put(16)
+    print(modin.config.NPartitions.get())
+
+    Output:
```

Comment:
This part is formatted a little strange in the docs. Can we just add a comment with the output in the line we call the print, for example:

```python
print(modin.config.Backend.get())  # prints 'Omnisci'
```

Reply:
Ok, in-line comments were added.